sympy/codegen/approximations.py

```diff
@@ -12,7 +12,10 @@
 """
 
 class SumApprox(Optimization):
-    """ Approximates sum by neglecting small terms
+    """ Approximates sum by neglecting small terms.
```

Comment:
These should be changed so the """ is on its own line.

Reply:
Changed.